test/chi/components/header.pug

```diff
@@ -165,7 +215,7 @@ h3 Header with Tabs and Buttons
           li
             a(href="#") Link
       .chi-header__end
-        a.chi-button.-flat(href="#") Button
+        a.chi-button.-flat.-dark(href="#") Button
         a.chi-button.-brand.-ml--1(href="#") Button
 
 h3 Header with Tabs, Icons and Buttons
```

Comment:
@sabah1910 please remove Header w/ tabs, icons and buttons example as it is the same example than Header w/ tabs